assets/src/scss/pages/_page.scss

```diff
@@ -64,12 +64,16 @@ div.page-template {
     margin-right: auto;
     display: block;
   }
+}
 
-  .single-campaign & {
+.single-campaign {
+  .page-template:not(.no-page-title) {
```

Comment:
Can you include this sunder `.page-template`? like `&:not(.no-page-title`

Reply:
@Inwerpsel done!